portfolio_ahardman_stylesheet.css

```diff
@@ -0,0 +1,339 @@
+/*   default settings   */
+:root{
+	--cyan_light: rgb(0, 204, 191);
+	--cyan_dark: rgb(0, 164, 153);
+}
+*{
+	box-sizing: border-box;
+	font-family: 'Segoe UI', Helvetica, Verdana; 
+}
+body{                 	/* removes default padding and margin around the ouside of the entire webpage */
+	font-size: 62.5%;
+	margin: 0;
+	padding: 0;
+	background-image: linear-gradient(to right,rgb(110,110,110),rgb(180,180,180));
+}
+@-ms-viewport{
+	width: device-width;
+}
+a{
+	text-decoration: none;
+	color:rgb(0,0,0);
+}
+input:focus{
+	border: none;
+	outline: none;
+}
+svg.logo{
+	z-index: -1;
+	position: absolute;
+	width: 100%;
+	height: 100%;
+	display: flex;
+	align-items: center;
+	background-image: linear-gradient(to right,rgb(110,110,110),rgb(180,180,180));
+}
+path{
+	stroke-width: 5px;
+	stroke: rgb(0, 204, 191);
+	fill: none;
+	stroke-linecap: round;
+}
+circle{
+	fill:rgb(0, 164, 153);
+}
+/* ----------------------navbar settings---------------------- */
+.navbar{
+	position:fixed;
+	position: sticky;
```

Comment:
position sticky actually does nothing in this context, just use fixed. will need to use `width: 100%;` to make it work